src/app/public/modules/text-editor/toolbar/text-editor-toolbar.component.html

```diff
@@ -230,6 +242,7 @@
             class="sky-btn sky-btn-default sky-btn-icon"
             title="Outdent"
             (click)="execCommand('outdent')"
+            [disabled]="disabled"
```

Comment:
Check your property order in this file. Input properties should come before outputs.

Reply:
fixed in "[correct property order](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/568099c6e41527039228cb0a6469beb1a426a35f)"